### DISCUSSION TITLE
Error message about Python during installation !!

### DISCUSSION THREAD

**Discussion by Alexbabylon** [bmaltais/kohya_ss#2574] [Q&A]
HI 
I am trying to install Kohya and keep give me error message regarding the python version ,although
<img width="862" alt="2024-06-08_7-04-15" src="https://github.com/bmaltais/kohya_ss/assets/128872360/a799c554-e1fa-4402-879e-2e9238a13e93">
<img width="1278" alt="2024-06-08_7-04-38" src="https://github.com/bmaltais/kohya_ss/assets/128872360/748e2193-70be-483f-a3d0-0876a9a9940d">
 i do have the correct version ..please any help appreciate it

**Comment by b-fission** [comment]
Can you try running `setup-3.10.bat`
